src/AutoMerge/util.jl

```diff
@@ -111,7 +111,8 @@ end
 
 function comment_text_pass(::NewVersion,
                            suggest_onepointzero::Bool,
-                           version::VersionNumber)
+                           version::VersionNumber,
+                           is_jll::Bool)
```

Comment:
this new argument doesn't seem to be used

Reply:
It's not but it's needed to allow a uniform call to both methods.

Reply:
https://github.com/JuliaRegistries/RegistryCI.jl/pull/407/files#diff-642068d5927ce10f0fca27aa4cc347618d5e8904fff6f21fcb2a542faa34cda3R210-R213 vs https://github.com/JuliaRegistries/RegistryCI.jl/pull/407/files#diff-642068d5927ce10f0fca27aa4cc347618d5e8904fff6f21fcb2a542faa34cda3R199-R202 look different still, where are they uniform?

Reply:
Those are different functions. It's the two methods of `comment_text_pass` that have been harmonized to allow a single call instead of the two different calls that were made from `new-package.jl` and `new-version.jl`.

Reply:
ahh ok, I get it now, makes sense!